src/pages/register-a-beacon/about-beacon-owner.tsx

```diff
@@ -0,0 +1,148 @@
+import { GetServerSideProps } from "next";
+import React, { FunctionComponent } from "react";
+import { BackButton, Button } from "../../components/Button";
+import { FormErrorSummary } from "../../components/ErrorSummary";
+import {
+  Form,
+  FormFieldset,
+  FormGroup,
+  FormLegendPageHeading,
+} from "../../components/Form";
+import { Grid } from "../../components/Grid";
+import { FormInputProps, Input } from "../../components/Input";
+import { Layout } from "../../components/Layout";
+import { IfYouNeedHelp } from "../../components/Mca";
+import { FormValidator } from "../../lib/formValidator";
+import { FormPageProps, handlePageRequest } from "../../lib/handlePageRequest";
+import { ensureFormDataHasKeys } from "../../lib/utils";
+
+const AboutBeaconOwner: FunctionComponent<FormPageProps> = ({
+  formData,
+  needsValidation,
+}: FormPageProps): JSX.Element => {
+  formData = ensureFormDataHasKeys(
+    formData,
+    "beaconOwnerFullName",
+    "beaconOwnerTelephoneNumber",
+    "beaconOwnerAlternativeTelephoneNumber",
+    "beaconOwnerEmail"
+  );
+  const pageHeading = "About the beacon owner";
+  const errors = FormValidator.errorSummary(formData);
+  const { beaconOwnerFullName, beaconOwnerEmail } = FormValidator.validate(
+    formData
+  );
+  const pageHasErrors = needsValidation && FormValidator.hasErrors(formData);
+
+  return (
+    <>
+      <Layout
+        navigation={
+          <BackButton href="/register-a-beacon/more-vessel-details" />
+        }
+        title={pageHeading}
+        pageHasErrors={pageHasErrors}
+      >
+        <Grid
+          mainContent={
+            <>
+              <Form action="/register-a-beacon/about-beacon-owner">
+                <FormFieldset>
+                  <FormErrorSummary
+                    showErrorSummary={needsValidation}
+                    errors={errors}
+                  />
+                  <FormLegendPageHeading>{pageHeading}</FormLegendPageHeading>
+
+                  <FullName
+                    value={formData.beaconOwnerFullName}
+                    showErrors={pageHasErrors && beaconOwnerFullName.invalid}
+                    errorMessages={beaconOwnerFullName.errorMessages}
+                  />
+
+                  <TelephoneNumber
+                    value={formData.beaconOwnerTelephoneNumber}
+                  />
+
+                  <AlternativeTelephoneNumber
+                    value={formData.beaconOwnerAlternativeTelephoneNumber}
+                  />
+
+                  <EmailAddress
+                    value={formData.beaconOwnerTownOrCity}
```

Comment:
@NyWeb The email address is not being set if a user submits the form and then navigates back to this page. Looks like there is a typo on the key, should be `beaconOwnerEmail` vs `beaconOwnerTownOrCity`, so:

```typescript
value={formData.beaconOwnerEmail}
```